src/core/game.js

```diff
@@ -0,0 +1,99 @@
+import Coords from './coords';
+
+export const EMPTY_VALUE = 0;
+
+export const buildGrid = (size = 4) => {
+    let value = EMPTY_VALUE;
+    let grid = [];
+
+    for (let y = 0; y < size; y++) {
+        grid[y] = [];
+
+        for (let x = 0; x < size; x++) {
+            value++;
+            grid[y][x] =
+                value == size * size ? EMPTY_VALUE : (grid[y][x] = value);
```

Comment:
The else part is weird: you assign the result of the assignation to your cell

Reply:
Prettier :(
I didn't see this weird part.